extensions/webjars-locator/deployment/src/main/java/io/quarkus/webjar/locator/deployment/WebJarLocatorStandaloneBuildStep.java

```diff
@@ -36,6 +38,8 @@ public void findWebjarsAndCreateHandler(
                     new RouteBuildItem(webjarRootPath + "*",
                             recorder.getHandler(webjarRootPath, webjarNameToVersionMap),
                             false));
+        } else {
+            log.warn("No WebJars were found in the project. Requests to the /webjars/ path will always return 404 (Not Found)");
```

Comment:
Do we really need this warning? It looks a bit too intrusive? I might misunderstand it but won't you have the warning as soon as you enable the extension?

Reply:
Intrusive? How so? The warning will only be displayed during build time if no webjar dependencies were are added to the build, which makes this extension useless.

Reply:
Sure but that's not how other extensions behave. You don't have a warn if you don't have any Hibernate ORM entities for instance.

Reply:
I see, perhaps it should display the encountered webjars then? Or do you want to completely remove it?